src/ClientData/CaptureData.cpp

```diff
@@ -129,16 +127,18 @@ void CaptureData::OnCaptureComplete(
         if (stats.count() > 0) {
           uint64_t elapsed_nanos = timer_info.end() - timer_info.start();
           int64_t deviation = elapsed_nanos - stats.average_time_ns();
-          stats.set_variance_ns(stats.variance_ns() + deviation * deviation);
+          id_to_sum_of_deviations_squared[timer_info.function_id()] += deviation * deviation;
         }
       }
     }
   }
 
-  for (auto& pair : functions_stats_) {
-    FunctionStats& stats = pair.second;
+  for (auto& pair : id_to_sum_of_deviations_squared) {
```

Comment:
nit. structured bindings can make the code more legible : `for (auto& [id, sum_of_deviation_squared] : id_to_sum_of_deviations_squared)`

Reply:
done. thanks!